system/modules/isotope/library/Isotope/Model/Gallery/Standard.php

```diff
@@ -364,12 +365,19 @@ protected function getImageForType($strType, array $arrFile, $blnWatermark = tru
             $picture = array('img'=>array('src'=>'', 'srcset'=>''), 'sources'=>array());
         }
 
+        $objResizedFile = new File(rawurldecode($strImage));
+
         // Watermark
         if ($blnWatermark
             && $this->{$strType . '_watermark_image'} != ''
             && ($objWatermark = \FilesModel::findByUuid($this->{$strType . '_watermark_image'})) !== null
         ) {
+            if(method_exists($objResizedFile, 'createIfDeferred')) {
```

Comment:
@ausi do we need to create the deferred size? I assume we don't need to if we just use `$objResizedFile->imageSize` to get the file size?

Reply:
Nevermind, I saw this was only the case when adding watermarks, where we obviously need the file 😆